mutations/events.py

```diff
@@ -773,6 +775,74 @@ def deleteEvent(eventid: str, info: Info) -> EventType:
     return EventType.from_pydantic(Event.model_validate(event_ref))
 
 
+@strawberry.mutation
+def rejectEvent(
+    eventid: str,
+    reason: str,
+    info: Info,
+) -> EventType:
+    """ Reject Event by CC and reset the state to incomplete """
+    user = info.context.user
+
+    if user is None or user["role"] not in ["cc"]:
+        raise Exception("Not Authenticated!")
+
+    query = {
+        "_id": eventid,
+    }
+    event_ref = eventsdb.find_one(query)
+    if event_ref is None:
+        raise Exception(
+            "Can not access event. Either it does not exist or user does not have perms."  # noqa: E501
+        )
+    event_instance = Event.model_validate(event_ref)
+
+    mail_uid = user["uid"]
+    clubDetails = getClubDetails(event_instance.clubid, info.context.cookies)
+    if len(clubDetails.keys()) == 0:
+        raise Exception("Club does not exist.")
+    else:
+        mail_club = clubDetails["email"]
+        clubname = clubDetails["name"]
+
+    if event_instance.status.state != Event_State_Status.pending_cc:
+        raise Exception("Cannot reset event that has progressed beyond CC.")
+
+    updation = {
```

Comment:
Update other values also, such as event submitted datetime, etc. Once just check it.